packages/cli/src/test/scripts/call.test.js

```diff
@@ -12,12 +16,32 @@ import CaptureLogs from '../helpers/captureLogs';
 import ProjectFile from '../../models/files/ProjectFile';
 import NetworkFile from '../../models/files/NetworkFile';
 
+import * as Compiler from '../../models/compiler/Compiler';
+import * as transpiler from '../../transpiler';
+
+const sandbox = sinon.createSandbox();
+
 describe('call script', function() {
   const [account] = accounts;
   const txParams = { from: account };
 
   const network = 'test';
 
+  beforeEach('stub getFromPathWithUpgradeable to simulate transpilation of contracts', async function() {
+    // stub getFromPathWithUpgradeable to fill upgradeable field with the same contract
```

Comment:
Should we refactor this bit into a helper, to avoid repetition?

Reply:
I'd say yes.